test/v2/controllers/requestParsers/RetrieveSelfAssessmentChargeHistoryRequestParserSpec.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.controllers.requestParsers
+
+import support.UnitSpec
+import v2.mocks.validators.MockRetrieveSelfAssessmentChargeHistoryValidator
+import v2.models.domain.Nino
+import v2.models.errors.{BadRequestError, ErrorWrapper, NinoFormatError, TransactionIdFormatError}
+import v2.models.request.retrieveSelfAssessmentChargeHistory.{RetrieveSelfAssessmentChargeHistoryRawData, RetrieveSelfAssessmentChargeHistoryRequest}
+
+class RetrieveSelfAssessmentChargeHistoryRequestParserSpec extends UnitSpec {
+
+  val validNino: String                    = "AA123456B"
+  val validTransactionId: String           = "717f3agW5415678f"
```

Comment:
Could we make this validTransactionId shorter to match the transactionId in the validatorSpec?